egov/egov-egi/src/main/resources/db/migration/main/V20170321115549__egi_wf_productisation_mylink_DDL.sql

```diff
@@ -0,0 +1,5 @@
+alter table eg_wf_states add mylinkid varchar(256);
+
+alter table eg_wf_states add tenantid varchar(250);
+
+alter table eg_wf_state_history add tenantid varchar(250);
```

Comment:
The script V20170321115549__egi_wf_productisation_mylink_DDL.sql is already removed by @manikanta-pt . Please check.